changelog.md

```diff
@@ -18,7 +32,6 @@
 
 #### Misc
 
-* `LayoutView#showChildView` new `options` argument passed to underlying `Region#show` to enable full `show` functionality.
```

Comment:
Turns out that didn't get in `2.4.3` though the PR was labeled that way it was only merged into `next`.  People were asking about it on gitter, so it was cherry-picked on to patch for `2.4.4`